frontend/src/components/NavBar/NavBar.jsx

```diff
@@ -97,24 +191,27 @@ export const NavBar = ({ user }) => {
           >
             <Button
               color='inherit'
-              aria-label='Alerts'
+              aria-label='Notifications'
               variant='text'
-              onClick={() => console.log('alert')}
+              onClick={() => (window.location.href = URL_INV_NOTIFICATIONS)}
             >
-              <StyledBadge badgeContent={0} color='error'>
+              <StyledBadge
+                badgeContent={notifications.numberOfNotifications}
+                color='error'
+              >
                 <NotificationsIcon style={{ marginRight: 5 }} />
               </StyledBadge>
-              Alerts
+              Notifications
             </Button>
             <Button
               color='inherit'
               aria-label='Cart'
               variant='text'
               onClick={(e) =>
                 isItemsPage
-                  ? navigate(URL_INV_CART)
+                  ? (window.location.href = URL_INV_CART)
```

Comment:
why do we prefer using `window.location.href`? should we stick with client side routing?